.github/workflows/build.yml

```diff
@@ -1574,3 +1574,39 @@ jobs:
 
     ##########################################################################################################################
     ##########################################################################################################################
+
+    ##########################################
+    #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~~/#########\~~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~|##|~~~~~|##|~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~|##|~~~~~|##|~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~|###########|~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~|##|~~~~~|##|~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~|##|~~~~~|##|~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~~\#########/~~~~~~~~~~~~~~#
+    #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+    # Stage Eight - Publish Artifacts        #
+    #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+
+    publish:
+        needs: [test_js_and_python, test_python, test_python_sdist]
+        if: startsWith(github.ref, 'refs/tags/v')
+        runs-on: ubuntu-22.04
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Install yarn
+              run: npm install -g yarn
+
+            - name: Install js dependencies
+              run: yarn --frozen-lockfile
+              env:
+                  PSP_SKIP_EMSDK_INSTALL: 1
+
+            - name: Run publish script
+              run: node tools/perspective-scripts/publish.mjs
+              env:
+                  CI: true
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  GITHUB_WORKFLOW_ID: ${{ github.run_id }}
+                  COMMIT: ${{ github.sha }}
```

Comment:
This isn't the commit sha, it's a flag to tell `publish.mjs` whether to run in dry-run mode. Setting this to SHA causes this script (in CI) to attempt to publish to pypi & npm, which it isn't authenticated for (and we don't want to do this for now also).